packages/dso-toolkit/gulp/build-svg-spritesheets.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-env node */
+
+import gulp from "gulp";
+import { basename } from "path";
+import { obj } from "list-stream";
+import { parse } from "css";
+import svgstore from "gulp-svgstore";
+import cheerio from "gulp-cheerio";
+import prettier from "gulp-prettier";
+import rename from "gulp-rename";
+
+import { sassTransformer } from "./transformers/sass.transformer.js";
+
+const iconsPath = "src/icons";
+const distPath = "dist";
+const outPutNameIcons = "dso-icons.svg";
+const outPutNameIconsWithVariants = "di.svg";
```

Comment:
Output is één woord. En outputName.. Klinkt als "filename"?

En ik zou door de file heen dezelfde taal spreken. Beide variabelen zijn één op één nodig voor de twee spritesheets:

```suggestion
const iconsSpritesheetFilename = "dso-icons.svg";
const diSpritesheetFilename = "di.svg";
```

Reply:
Ah. Als je deze suggustie overneemt, dan zijn we er ook. Ik resolve mijn review opmerking over hoofdletter P

Reply:
Ik heb deze wijziging doorgevoerd